.github/workflows/tests-main.yml

```
@@ -11,7 +11,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        python-version: ['3.9', '3.10', '3.11']
+        python-version: ['3.8', '3.9', '3.10', '3.11']
```

Python 3.8 is set for EOL in Oct 2024, so I don't think we should include it in our tests: https://devguide.python.org/versions/



```suggestion
        python-version: ['3.9', '3.10', '3.11']
```